test_functions.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import functions
+
+def test_readStateFile(tmp_path):
+    # Create a temporary file with test data
+    test_file = tmp_path / "test.txt"
+    test_file.write_text("Test data")
+
+    # Call the function with the test file
+    result = functions.readStateFile(test_file)
+
+    # Assert that the result is equal to the expected output
+    assert result == ["Test data"]

```

Comment:
Let's do something more realistic for this test.
Like a line from a actual state test file.